### PR TITLE
cleanup variables

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -389,14 +386,14 @@ fn send_detach_message(unix_stream_stdin: &mut UnixStream) -> anyhow::Result<()>
 #[logfn(Debug)]
 #[logfn_inputs(Debug)]
 fn run_client(unix_stream: &mut UnixStream, escape_code: u8) -> anyhow::Result<()> {
-    let mut _stdout = stdout().into_raw_mode()?;
```
Comment (errordeveloper) [inline src/lib.rs:393]: well, so `_stdout` was unused, however it looks like `into_raw_mode()` has a desired side-effect?

Comment (yazgoo) [reply src/lib.rs:393]: yes, correct

Comment (errordeveloper) [reply src/lib.rs:393]: I've changed it to use just one variable and make the notion more explicit and it's working.

**Comment by errordeveloper** [comment]
@yazgoo this should be good to merge!

**Pull request merged**